daprdocs/content/en/operations/resiliency/resiliency-overview.md

```diff
@@ -0,0 +1,169 @@
+description: "Configure Dapr error retries, timeouts, and circuit breakers"
+---
+
+> Resiliency is currently a preview feature. Before you can utilize resiliency policies, you must first [enable the resiliency preview feature]({{< ref preview-features >}}).
+
+## Introduction
+
+Distributed applications are commonly comprised of many moving pieces, with dozens, even hundreds, of instances for any given service. With so many moving pieces, the likelihood of a system failure increases. For example, an instance can fail due to hardware failures, an overwhelming number of requests, application restarts/scale outs, or any other reason. These events can cause a network call between services to fail. Designing and implementing your application with fault tolerance (the ability to detect, mitigate, and respond to failures) allows your application to recover quickly to a functioning state.
+
+## Overview
+
+Dapr provides a mechanism for defining and applying fault tolerance/resiliency policies via a [resiliency spec]({{< ref "resiliency-overview.md#complete-example-policy" >}}). The resiliency spec is defined in the same location as components and is applied when the Dapr sidecar starts. The sidecar determines when and how to apply resiliency policies to your Dapr API calls. Within the resiliency spec, you define policies for popular resiliency patterns, such as:
+
+- [Timeouts]({{< ref "policies.md#timeouts" >}})
+- [Retries/back-offs]({{< ref "policies.md#retries" >}})
+- [Circuit breakers]({{< ref "policies.md#circuit-breakers" >}})
+
+Policies can then be applied consistently to [targets]({{< ref "targets.md" >}}), which include:
+
+- [Apps]({{< ref "targets.md#apps" >}}) via service invocation
+- [Components]({{< ref "targets.md#components" >}})
+- [Actors]({{< ref "targets.md#actors" >}})
+
+Additionally, resiliency policies can be [scoped to specific apps]({{< ref "component-scopes.md#application-access-to-components-with-scopes" >}}).
+
+Below is the general structure of what a resiliency policy looks like:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Resiliency
+metadata:
+  name: resiliency
+scopes:
+  # optionally scope the policy to specific apps
+spec:
+  policies:
+    timeouts:
+      # timeout policy definitions
+
+    retries:
+      # retry policy definitions
+
+    circuitBreakers:
+      # circuit breaker policy definitions
+
+  targets:
+    apps:
+      # apps and their applied policies here
+
+    actors:
+      # actor types and their applied policies here
+
+    components:
+      # components and their applied policies here
+```
+
+> Note: In self-hosted mode, the resiliency policy must be named `resiliency.yaml` and reside in the components folder provided to the sidecar. In Kubernetes, Dapr scans all resiliency policies.
+
+### Complete Example Policy
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Resiliency
+metadata:
+  name: resiliency
+# simialrly to Subscriptions CRD, scopes lists the Dapr App IDs that this
```

Comment:
This is a confusing comment. Configurations can also be scoped. And we should use the word "spec" and not CRD. Make this consistent. 

```suggestion
# simialrly to Subscriptions CRD, scopes lists the Dapr App IDs that this
```